webrtc-stats.html

```diff
@@ -1275,6 +1276,16 @@ <h2>
                   Calculated as defined in [[!RFC7002]] section 3.2 and Appendix A.a.
                 </p>
               </dd>
+              <dt>
+                <dfn>fecBytesReceived</dfn> of type <span class=
+                "idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP FEC bytes received for this <a>SSRC</a>, only including payload bytes.
+                  This is a subset of {{RTCInboundRtpStreamStats/bytesReceived}}.
```

Comment:
This looks good to me. What about fecHeaderBytesReceived as a subset of headerBytesReceived for completeness or is there something about that that doesn't make sense?

Reply:
Ack